packages/nuxt/src/app/composables/cookie.ts

```diff
@@ -27,7 +27,7 @@ const CookieDefaults: CookieOptions<any> = {
   encode: val => encodeURIComponent(typeof val === 'string' ? val : JSON.stringify(val))
 }
 
-export function useCookie <T = string | null> (name: string, _opts?: CookieOptions<T>): CookieRef<T> {
+export function useCookie <T = string | undefined> (name: string, _opts?: CookieOptions<T>): CookieRef<T> {
```

Comment:
Thank you! Though, let's preserve backwards compatibility to allow people to assign `null` to the ref.

```suggestion
export function useCookie <T = string | null | undefined> (name: string, _opts?: CookieOptions<T>): CookieRef<T> {
```